interface/patient_file/summary/dashboard_header.php

```diff
@@ -0,0 +1,135 @@
+<?php
+  /**
+ * Dash Board Header.
+ *
+ * @package   OpenEMR
+ * @link      http://www.open-emr.org
+ * @author Ranganath Pathak <pathak@scrs1.org>
+ * @copyright Copyright (c) 2018 Ranganath Pathak <pathak@scrs1.org>
+ * @version 1.0.0
+ * @license   https://github.com/openemr/openemr/blob/master/LICENSE GNU General Public License 3
+ */
+ 
+if ($GLOBALS['enable_help'] == 1) {
+    $help_icon = '<a class="oe-pull-away oe-help-redirect" data-target="#myModal" data-toggle="modal" href="#" id="help-href" name="help-href" style="color:#676666" title="' . xla("Click to view Help") . '"><i class="fa fa-question-circle" aria-hidden="true"></i></a>';
+} elseif ($GLOBALS['enable_help'] == 2) {
+    $help_icon = '<a class="oe-pull-away oe-help-redirect" data-target="#myModal" data-toggle="modal" href="#" id="help-href" name="help-href" style="color:#DCD6D0 !Important" title="' . xla("To enable help - Go to  Administration > Globals > Features > Enable Help Modal") . '"><i class="fa fa-question-circle" aria-hidden="true"></i></a>';
+} elseif ($GLOBALS['enable_help'] == 0) {
+    $help_icon = '';
+}
+?>
+
+<div class="page-header clearfix">
+    <h2 id="header_title" class="clearfix"><span id='header_text'><?php echo attr($header_title)?><?php echo " " . text(getPatientNameFirstLast($pid));?></span><?php echo $help_icon; ?></h2>
+
+<?php
+// If patient is deceased, then show this (along with the number of days patient has been deceased for)
+$days_deceased = is_patient_deceased($pid);
```

Comment:
I am pretty sure the algorithm for displaying deceased message this has changed recently. Rec. checking the most recent codebase on this.

Reply:
I reworked the recently changed code to make it more meaningful because 'Recently deceased' did not convey any meaningful information. Now it gives a date and number of days till 90 then number of months till 24 and then number of years thereafter:)